### DISCUSSION TITLE
win11升级最新版本后，边缘触发调音量造成explorer卡死重启

### DISCUSSION THREAD

**Discussion by Victorjrc** [shuax/MouseInc.Settings#224] [Q&A] [answered]
win11升级最新版本后，边缘触发调音量造成explorer卡死重启
恢复到2.11.8版本也一样，不知道其他人是否遇到？
回去试验其他PC后再更新

**Comment by shuax** [comment, marked as answer]
系统bug，我看别人也是这样，调整音量，资源管理器崩溃。

**Reply by Victorjrc** [reply]
另一个台式机就没有出现这个情况，应该是win11的问题，最近家人的本子出现开机无法进系统，感觉是win11最近的更新bug

**Reply by Victorjrc** [reply]
这种就只能重装或者等系统更新了吗？有别的办法么

**Comment by Victorjrc** [comment]
更新下，win11这周更新到22538版本后，bug消失，而且调整音量会在任务栏上方中间位置出现调节菜单
